docs/source/markdown/podman.1.md

```diff
@@ -422,7 +422,7 @@ See the `subuid(5)` and `subgid(5)` man pages for more information.
 
 Images are pulled under `XDG_DATA_HOME` when specified, otherwise in the home directory of the user under `.local/share/containers/storage`.
 
-Currently the slirp4netns package is required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
+Currently either slirp4netns or pasta are required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
```

Comment:
model to previous comments

Reply:
In which way, here? Your comments clearly apply to `podman-create.1.md.in` and `podman-run.1.md.in`, but I don't see any issue in `podman.1.md`.

Reply:
Oh, I see now, the verb doesn't agree with the closest noun in _or_. Changed, and dropped _either_, as it's not really fundamental here.